app.ts

```diff
@@ -0,0 +1,10 @@
+import express from 'express'
+import * as bodyparser from 'body-parser'
+import requestLoggerMiddleware from "./request.logger.middleware";
+const app =  express()
+
+// middleware
+app.use(bodyparser.json())
+app.use(requestLoggerMiddleware)
```

Comment:
only do this on DEBUG env variable is set to true
```suggestion
const DEBUG = process.env.DEBUG == 'true'
if (DEBUG) {
  app.use(requestLoggerMiddleware)
}
```